src/backend/expungeservice/models/charge_types/sex_crimes.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+
+from expungeservice.models.charge import Charge
+from expungeservice.models.expungement_result import TypeEligibility, EligibilityStatus
+
+
+@dataclass(eq=False)
+class SexCrimes(Charge):
+    type_name: str = "Sex Crimes"
+    expungement_rules: str = (
+        """
+        Sex Crimes are type-ineligible for expungement, other than a narrow exception for "Romeo and Juliet" cases.
+        According to 137.225(6)(a):
+        Notwithstanding subsection (5) of this section, the provisions of subsection (1)(a) of this section do not apply 
```

Comment:
I would leave out original statute text here complete text and just have a summarizing note: 
"The required eligibility conditions for this exception are listed in subsection 137.225(6)(a)."

Reply:
good call, changed